app/assets/javascripts/student_profile/help_bubble.js

```diff
@@ -7,11 +7,13 @@
   var HelpBubble = window.shared.HelpBubble = React.createClass({
     propTypes: {
       title: React.PropTypes.string.isRequired, // e.g. 'What is a Note?'
-      content: React.PropTypes.string.isRequired, // HTML string which will be rendered in the modal's box.
+      content: React.PropTypes.object.isRequired, // React DOM objects which will be displayed in the modal text box.
```

Comment:
FYI, you can use `React.PropTypes.element` to indicate it has to be a React element like your good comment says :)